js/gui.js

```diff
@@ -63,27 +63,31 @@ var keyHandlers = {
     "ArrowLeft": function() {
         swal.close();
         if (player.canMoveLeft()) {
+            stopAudio();
```

Comment:
Please have a look here:
https://github.com/fossasia/labyrinth/blob/master/js/player.js#L14
If we put this into 
https://github.com/fossasia/labyrinth/blob/master/js/tiles.js#L112
PlacedTile.prototype.playerLeaves
I am a 100% satisfied as there is no code duplication.